### PR TITLE
Partial Filipino Translation - 1 187 translated words 

### PR DESCRIPTION
**Pull request by mind-blast**
Partial Filipino Translation for recording-changes.asc .

Code text and computer generated code text are not counted as translated words.

### CODE REVIEW

File: book/02-git-basics/sections/recording-changes.asc [hunk 1]
```diff
@@ -212,24 +212,24 @@ $ cat .gitignore
 *~
 ----
 
-The first line tells Git to ignore any files ending in ``.o'' or ``.a'' -- object and archive files that may be the product of building your code.
-The second line tells Git to ignore all files whose names end with a tilde (`~`), which is used by many text editors such as Emacs to mark temporary files.
-You may also include a log, tmp, or pid directory; automatically generated documentation; and so on.
-Setting up a `.gitignore` file for your new repository before you get going is generally a good idea so you don't accidentally commit files that you really don't want in your Git repository.
+Ang unang linya ay nagsasabi ng Git na hindi pansinin ang anumang mga file na nagtatapos sa ``.o'' o ``.a'' -- na bagay at archive na mga file na maaari ang produkto ng pag-build ng iyong code.
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: I think the word `object` is a file type here. Hence, should not be translated.

File: book/02-git-basics/sections/recording-changes.asc [hunk 2]
```diff
@@ -212,24 +212,24 @@ $ cat .gitignore
...
+Ang pangalawang linya ay nagsasabi ng Git na hindi pansinin ang lahat ng mga file na kaninong mga pangalan ay nagtatapos na may (`~`), na ginagamit ng maraming mga tekstong editor tulad ng Emacs upang markahan ang pansamantalang mga file.
+Maaari mo ring magsama ng log, tmp, o pid na direktoryo; awtomatikong nabuo na automatically generated dokumentasyon; at iba pa.
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: please remove `automatically generated`

File: book/02-git-basics/sections/recording-changes.asc [hunk 3]
```diff
@@ -334,10 +334,10 @@ index 0000000..03902a1
 +My Project
 ----
 
-It's important to note that `git diff` by itself doesn't show all changes made since your last commit -- only changes that are still unstaged.
-If you've staged all of your changes, `git diff` will give you no output.
+Ito ay importanteng i-note na ang `git diff` na sa sarili niya ay hindi nagpapakita ng lahat ng mga pagbabago na iyong nagawa sa huli mong commit -- mga pagbabago lamang na hindi pa rin naitanghal.
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: you could change `Ito ay importanteng i-note` to `Importanteng tandaan`

**Review by josteinfrancois — COMMENTED** [review]
Hello @robin-ho. I have reviewed your work. Please refer to the comments I've provided. Thank you.

File: book/02-git-basics/sections/recording-changes.asc (hunk 1 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: one change.

File: book/02-git-basics/sections/recording-changes.asc (hunk 2 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change.

File: book/02-git-basics/sections/recording-changes.asc (hunk 3 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

**Comment by mind-blast** [comment]
Done changing all suggested words. Thanks for checking them out.

File: book/02-git-basics/sections/recording-changes.asc [hunk 4]
```diff
@@ -359,7 +359,7 @@ Changes not staged for commit:
     modified:   CONTRIBUTING.md
 ----
 
-Now you can use `git diff` to see what is still unstaged:
+Ngayon ay maaari mong gamitin ang `git diff` para tingnan kung ano pa ang hindi naitanghal:
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/recording-changes.asc]: maaari mo nang gamitin

File: book/02-git-basics/sections/recording-changes.asc [hunk 5]
```diff
@@ -396,33 +396,33 @@ index 8ebb991..643e24f 100644
...
-[NOTE]
-.Git Diff in an External Tool
+[TANDAAN]
+.Ang Git Diff ay isang Panlabas na Kasangkapan
 ====
-We will continue to use the `git diff` command in various ways throughout the rest of the book.
-There is another way to look at these diffs if you prefer a graphical or external diff viewing program instead.
-If you run `git difftool` instead of `git diff`, you can view any of these diffs in software like emerge, vimdiff and many more (including commercial products).
-Run `git difftool --tool-help` to see what is available on your system.
+Patuloy kaming gagamit ng `git diff` na utos sa iba't ibang paraan sa buong pahina ng aklat.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/recording-changes.asc]: Patuloy tayong gagamit

File: book/02-git-basics/sections/recording-changes.asc [hunk 6]
```diff
@@ -396,33 +396,33 @@ index 8ebb991..643e24f 100644
...
 [[_committing_changes]]
-==== Committing Your Changes
+==== Pag-commit ng iyong mga Binago
 
-Now that your staging area is set up the way you want it, you can commit your changes.
-Remember that anything that is still unstaged -- any files you have created or modified that you haven't run `git add` on since you edited them -- won't go into this commit.
-They will stay as modified files on your disk.
-In this case, let's say that the last time you ran `git status`, you saw that everything was staged, so you're ready to commit your changes.(((git commands, status)))
-The simplest way to commit is to type `git commit`:(((git commands, commit)))
+Ngayon na ang iyong itinanghal na lugar ay nai-set sa paraan na gusto mo, maaari mong i-commit ang iyong mga binago.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/recording-changes.asc]: nai-setup ayon sa gusto mo

File: book/02-git-basics/sections/recording-changes.asc [hunk 7]
```diff
@@ -396,33 +396,33 @@ index 8ebb991..643e24f 100644
...
+Ang pinakasimpleng paraan para ma-commit ay mag-type ng `git commit`:(((git commands, commit)))
 
 [source,console]
 ----
 $ git commit
 ----
 
-Doing so launches your editor of choice.
-(This is set by your shell's `EDITOR` environment variable -- usually vim or emacs, although you can configure it with whatever you want using the `git config --global core.editor` command as you saw in <<_getting_started#_getting_started>>).(((editor, changing default)))(((git commands, config)))
+Ang paggawa nito ay naglulunsad ng iyong editor ng pagpipilian.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/recording-changes.asc]: editor na napili.

File: book/02-git-basics/sections/recording-changes.asc (hunk 4 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change.

File: book/02-git-basics/sections/recording-changes.asc (hunk 5 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 6 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 7 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

**Comment by mind-blast** [comment]
Done change all suggested words @severinolorillajr severinolorillajr

**Pull request merged**